docs/source/get_involved/governance.rst

```diff
@@ -400,9 +400,11 @@ core developers.
    Ten working days excluding week-ends constitute "reasonable time" in the above sense.
    The period resets at every new change made to the PR.
    It starts only when all GitHub checks pass.
--  During this period, the PR can be merged if it has an approval and no rejection, but should be
-   reverted if it receives a rejection in addition.
--  If the "reasonable time" period elapses and no approval or rejection has been expressed on a PR,
+-  During this period, the PR can be merged if it has an approval and no rejection after two working days excluding week-ends,
+   or if five working days excluding week-ends have passed (without any additional condition).
```

Comment:
Perhaps "without any additional condition" is a bit strong, maybe a rejection should halt this 🙂. That would be the case currently, correct?

Reply:
that's not what I meant, I meant if there is no review activity, and it's authored by a core dev, it can be merged.